it_IT/docs/recipes/when-to-use-plan.md

```diff
@@ -0,0 +1,139 @@
+___
+**Nota del traduttore**
+
+Questa è la traduzione del file [readme.md](https://github.com/sindresorhus/ava/blob/master/readme.md). Qui c'è il [link](https://github.com/sindresorhus/ava/compare/f2c070987ecee3caf7613190acf2c8a90700e058...master#diff-f2c070987ecee3caf7613190acf2c8a90700e058) dove si confrontano le differenze tra commit di riferimento di questa traduzione e l'ultimo commit di AVA sulla branch master (Se si clicca sul link, e non si vede il file `readme.md` nella lista dei file modificati, questa traduzione è aggiornata).
+___
+
+# Quando usare `t.plan()`
+
+Traduzioni: [Español](https://github.com/sindresorhus/ava-docs/blob/master/es_ES/docs/recipes/when-to-use-plan.md), [Français](https://github.com/sindresorhus/ava-docs/blob/master/fr_FR/docs/recipes/when-to-use-plan.md), [日本語](https://github.com/sindresorhus/ava-docs/blob/master/ja_JP/docs/recipes/when-to-use-plan.md),  [Português](https://github.com/sindresorhus/ava-docs/blob/master/pt_BR/docs/recipes/when-to-use-plan.md), [Русский](https://github.com/sindresorhus/ava-docs/blob/master/ru_RU/docs/recipes/when-to-use-plan.md)
+
+Una delle differenze sostanziali tra AVA e [`tap`](https://github.com/tapjs/node-tap)/[`tape`](https://github.com/substack/tape) è il comportamento della funzione `t.plan()`. In AVA, `t.plan()` è solamente usato per verificare che il numero previsto i asserzioni sia rispettato, ma non termina automaticamente il test.
+
+## Uso superficiale di `t.plan()`
+
+Molti utenti venendo da `tap`/`tape` sono abituati ad usare `t.plan()` abbondantemente in ogni test. In ogni caso, in AVA, non viene considerato una "buona pratica". Invece `t.plan()` dovrebbe essere usato in scenari dove potrebbe realmente aggiungere valore al test.
+
+### Test sincroni senza ramificazioni
+
+Non è necessario usare `t.plan()` nella maggior parte dei test sincroni.
+
+```js
+test(t => {
+	// Sbagliato: non c'è alcuna ramificazione qui - t.plan() non è utile
+	t.plan(2);
+
+	t.is(1 + 1, 2);
+	t.is(2 + 2, 4);
+});
+```
+
+`t.plan()` non fornisce alcun valore aggiunto in questo caso, ma aggiunge solo difficoltà se volessi decidere di aggiungere o rimuovere un'asserzione.
+
+### Promesse che ci sia aspetta siano risolte
+
+```js
+test(t => {
+	t.plan(1);
+
+	return somePromise().then(result => {
+		t.is(result, 'foo');
+	});
+});
+```
+
+Ad una prima occhiata questo test sembra usare giustamente `t.plan()` dato che un handler per una promessa asincrona è conionvolto. Ci sono però alcuni problemi con il test:\
```

Comment:
[sinceramente userei sempre "promise". Ma se anche in altre documentazioni italiane si usa "promessa" allora ok..